package.json

```diff
@@ -26,7 +26,7 @@
     "dependencies": {
         "gray-matter": "4.0.2",
         "js-yaml": "3.13.1",
-        "jsdom": "^15.1.1",
+        "jsdom": "15.2.0",
```

Comment:
I don't like ranges but this time I got the range notation removed because I used `npm install --save jsdom@15.2.0`.